app/models/miq_server/configuration_management.rb

```diff
@@ -2,16 +2,11 @@ module MiqServer::ConfigurationManagement
   extend ActiveSupport::Concern
   include ConfigurationManagementMixin
 
-  def get_config(type = "vmdb")
-    if is_local?
-      VMDB::Config.new(type)
-    else
-      VMDB::Config.for_resource(type, self)
-    end
+  def config
+    (is_local? ? ::Settings : Vmdb::Settings.for_resource(self)).to_hash
```

Comment:
Should this be `::Settings.to_hash`?

Reply:
WE call `to_hash` a few characters to the right.

Reply:
Ah... I didn't see the outer parens wrapped both.